test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js

```diff
@@ -3,18 +3,20 @@ define([
     'kbase/js/widgets/narrative_core/upload/stagingAreaViewer',
     'base/js/namespace',
     'kbaseNarrative',
-], ($, StagingAreaViewer, Jupyter) => {
+    'narrativeConfig',
+], ($, StagingAreaViewer, Jupyter, Narrative, Config) => {
     'use strict';
 
     describe('The staging area viewer widget', () => {
         let stagingViewer, container, $container, $parentNode;
         const startingPath = '/',
-            updatePathFn = null, //() => {},
-            fakeUser = 'notAUser';
+            updatePathFn = null,
+            fakeUser = 'notAUser',
+            stagingServiceUrl = Config.url('staging_api_url');
 
         beforeEach(() => {
             jasmine.Ajax.install();
-            jasmine.Ajax.stubRequest(/.*\/staging_service\/list\/?/).andReturn({
```

Comment:
Set this to be configured, as I was running a Staging Service locally and this was trying to pass calls through!